dojo/api.py

```diff
@@ -89,7 +95,85 @@ def form_args(self, bundle):
         return kwargs
 
 
+def sanitize(text):
+    # We put the single quotes back, due to their frequent usage in exception
+    # messages.
+    return escape(text).replace('&#39;', "'").replace('&quot;', '"').replace('&#x27;', "'")
+
+
 class BaseModelResource(ModelResource):
+
+    def wrap_view(self, view):
+        """
+        Wraps views to check if APIv1 is enabled
+        """
+        @csrf_exempt
+        def wrapper(request, *args, **kwargs):
+            try:
+                api_v1_deprecation_warning = 'APIv1 is deprecated and may contain vulnerabilities. It\'s disabled by default. '
```

Comment:
If we change the message to:
APIv1 is deprecated and may contain vulnerabilities. **It is** disabled by default.

Doesn't that remove the need for lines 98 to 101 above?  I'd rather remove conjunctions (don't, can't, ...) from our messages + remove that character as accepted vs have conjunctions and use the sanitize function.  Keeping things simple :wink:   

And yes, I see the irony of me using multiple conjunctions in this comment. :shrug:

Reply:
I would agree and I always think that writing with full words and no contraction is certainly better. But we are bound to have some. Maybe move it to a general helper instead though? (and put `it is` ;-) )

Reply:
The sanitize is there from tasty pie itself, didn't want to change it. And there can be other errors from normal api operations. I think in general escaping is not needed for json api, but easiest is to stick what tasty pie does (the code is copied from there)
Would be bad usually. But this will be thrown away in june.

Reply:
Can look at avoiding conjunctions.

Reply:
@mtesauro let me know if it is acceptable now :-)

Reply:
sanitize removed and import from tastypie